models/comment.models.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+const { selectArticleById } = require("./article.models");
+
+exports.insertComment = (article_id, username, body) => {
+  return selectArticleById(article_id).then((articleExists) => {
+    if (!articleExists) {
+      res.status(404).send({ msg: "Article does not exist" });
```

Comment:
Again use Promise.rejects so that your errors are handled by your error handlers.  Additionally requests and responses are meant to be dealt with in the controller.   Most of the time, think res and req in the controller,  and checking for bad requests / not founds and responding with Promise.rejects as belonging in the models.